common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java

```diff
@@ -305,6 +305,11 @@ public class RssBaseConf extends RssConf {
           .stringType()
           .defaultValue("8k")
           .withDescription("The buffer size to cache the write index content for LOCALFILE.");
+  public static final ConfigOption<String> RSS_STORAGE_LOCALFILE_WRITER_CLASS =
+      ConfigOptions.key("rss.storage.localfile.writer.class")
```

Comment:
Could you use camel style for last segement?

Reply:
I renamed to `rss.storage.localFileWriter.class`.